global-styles/github-markdown.scss

```diff
@@ -73,939 +13,993 @@
+    font-weight: inherit;
+    font-weight: bolder;
+  }
 
-.markdown-body * {
-  box-sizing: border-box;
-}
+  h1 {
+    font-size: 2em;
+    margin: .67em 0;
+  }
 
-.markdown-body input {
-  font-family: inherit;
-  font-size: inherit;
-  line-height: inherit;
-}
+  img {
+    border-style: none;
+  }
 
-.markdown-body a {
-  color: #0366d6;
-  text-decoration: none;
-}
+  code,
+  kbd,
+  pre {
+    font-family: monospace, monospace;
+    font-size: 1em;
+  }
 
-.markdown-body a:hover {
-  text-decoration: underline;
-}
+  hr {
+    box-sizing: initial;
+    height: 0;
+    overflow: visible;
+  }
 
-.markdown-body strong {
-  font-weight: 600;
-}
+  input {
+    font: inherit;
+    margin: 0;
+  }
 
-.markdown-body hr {
-  height: 0;
-  margin: 15px 0;
-  overflow: hidden;
-  background: transparent;
-  border: 0;
-  border-bottom: 1px solid #dfe2e5;
-}
+  input {
+    overflow: visible;
+  }
 
-.markdown-body hr:after,
-.markdown-body hr:before {
-  display: table;
-  content: "";
-}
+  [type=checkbox] {
+    box-sizing: border-box;
+    padding: 0;
+  }
 
-.markdown-body hr:after {
-  clear: both;
-}
+  * {
+    box-sizing: border-box;
+  }
 
-.markdown-body table {
-  border-spacing: 0;
-  border-collapse: collapse;
-}
+  input {
+    font-family: inherit;
+    font-size: inherit;
+    line-height: inherit;
+  }
 
-.markdown-body td,
-.markdown-body th {
-  padding: 0;
-}
+  a {
+    color: #0366d6;
+    text-decoration: none;
+  }
 
-.markdown-body details summary {
-  cursor: pointer;
-}
+  a:hover {
+    text-decoration: underline;
+  }
 
-.markdown-body kbd {
-  display: inline-block;
-  padding: 3px 5px;
-  font: 11px SFMono-Regular, Consolas, Liberation Mono, Menlo, monospace;
-  line-height: 10px;
-  color: #444d56;
-  vertical-align: middle;
-  background-color: #fafbfc;
-  border: 1px solid #d1d5da;
-  border-radius: 3px;
-  box-shadow: inset 0 -1px 0 #d1d5da;
-}
+  strong {
+    font-weight: 600;
+  }
 
-.markdown-body h1,
-.markdown-body h2,
-.markdown-body h3,
-.markdown-body h4,
-.markdown-body h5,
-.markdown-body h6 {
-  margin-top: 0;
-  margin-bottom: 0;
-}
+  hr {
+    height: 0;
+    margin: 15px 0;
+    overflow: hidden;
+    background: transparent;
+    border: 0;
+    border-bottom: 1px solid #dfe2e5;
+  }
 
-.markdown-body h1 {
-  font-size: 32px;
-}
+  hr:after,
+  hr:before {
+    display: table;
+    content: "";
+  }
 
-.markdown-body h1,
-.markdown-body h2 {
-  font-weight: 600;
-}
+  hr:after {
+    clear: both;
+  }
 
-.markdown-body h2 {
-  font-size: 24px;
-}
+  table {
+    border-spacing: 0;
+    border-collapse: collapse;
+  }
 
-.markdown-body h3 {
-  font-size: 20px;
-}
+  td,
+  th {
+    padding: 0;
+  }
 
-.markdown-body h3,
-.markdown-body h4 {
-  font-weight: 600;
-}
+  details summary {
+    cursor: pointer;
+  }
 
-.markdown-body h4 {
-  font-size: 16px;
-}
+  kbd {
+    display: inline-block;
+    padding: 3px 5px;
+    font: 11px SFMono-Regular, Consolas, Liberation Mono, Menlo, monospace;
```

Comment:
Do we want to insert the same monospaced font from the editor here?

Reply:
Nope. It's override by another file.

Reply:
Okay